src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -2247,12 +2250,3 @@
 type: group
 short-summary: Manage shared access policies of a storage table.
 """
-
```

Comment:
why removing command group description here?

Reply:
> why removing command group description here?

This was copied from 'storage table policy' whose summary not suitable for 'storage queue'.
Actually we already have 'storage queue' description from line 2023 to line 2026

https://github.com/Azure/azure-cli/pull/15494/files/144c774e1877079a962235efe30cb3bcc233b971#diff-d6b7a29513fb666c1dee4ee655c8af9ab67ab76066278ad4ebc73a457a970d88#L2023-L2026